src/tools/crossgen2/Common/TypeSystem/CodeGen/MethodDesc.CodeGen.cs

```diff
@@ -57,6 +57,14 @@ public virtual bool IsAggressiveOptimization
             }
         }
 
+        public virtual bool RequireSecObject
```

Comment:
Can you add a comment for what this does? Core type system properties are pretty religious about IntelliSense.